invokeai/app/api/routers/images.py

```diff
@@ -83,7 +83,7 @@ async def get_thumbnail(
     status_code=201,
 )
 async def upload_image(
-    file: UploadFile, request: Request, response: Response
+    file: UploadFile, image_type: ImageType, request: Request, response: Response
```

Comment:
Allowing user to specify type is very risky.

Reply:
think the options would be then:
- just make the intermediates `uploads`, but then that kinda defeats the purpose
- add another route for uploading intermediates

any preference or other ideas? the use-case is putting canvas init/mask images in the `intermediates` folder

Reply:
let's use this issue #3385 to figure out a better way to handle this